jekyll/_cci2/contexts.md

```diff
@@ -254,6 +264,21 @@ endpoint with the appropriate request body. For this request, replace the
 environment variable name. The request body should include a `value` key 
 containing the plaintext secret as a string.
 
+##### Environment Variable validation
+
+###### Environment variable names
+
+The environment variable name must be 300 or fewer characters, must have alpha or _ as the first character and can contain alphanumeric or _ as the remaining characters.
```

Comment:
and can *only* contain

Reply:
Can we also wrap underscores in in `backticks` for easier reading?